lib/testing/test_runner.sh

```diff
@@ -251,6 +251,28 @@ get_vlan_prefix()
     (IFS=:; set -- $prefix; printf "%s:%s:%s:%x::" "$1" "$2" "$3" $(($4 + $vid * 4096)))
 }
 
+check_python()
+{
+    if [ -z ${PYTHON+unset} ]; then
```

Comment:
What's the `+unset`?

Just `if [[ -z "$PYTHON" ]]` ?

(Please use double-brace tests for all string tests, we're running this through bash deliberately so we don't have to dump down to /bin/sh syntax)

Reply:
I used the substitution since `nounset` is set and so `if [[ -z "$PYTHON" ]]` gives unbound variable error. And I used single brackets because they were used in the rest of the file, but I will use double brackets instead.

Reply:
Ah, I see... The fact that you can omit the colon is buried somewhat in the bash man page.

However, in this case I do actually think we'd want to match on both unset and null, so maybe `if [[ -z "${PYTHON:-}" ]]` is better?